<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import from Image</string>
  </property>
  <widget class="QPushButton" name="btnImport">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbMode">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>21</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ByTransparency</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ByTransparency (50%)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ByColor</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ByColor (Reverse)</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spnWidth">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>61</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnHeight">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>91</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </widget>
  <widget class="MyGraphicsArea" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="viewportUpdateMode">
    <enum>QGraphicsView::FullViewportUpdate</enum>
   </property>
  </widget>
  <widget class="IndicatorButton" name="chkAspectRatio">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>75</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="uires.qrc">
     <normaloff>:/link</normaloff>:/link</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>btnImport</zorder>
  <zorder>cmbMode</zorder>
  <zorder>spnWidth</zorder>
  <zorder>spnHeight</zorder>
  <zorder>btnOpen</zorder>
  <zorder>chkAspectRatio</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsArea</class>
   <extends>QGraphicsView</extends>
   <header>mygrar.h</header>
  </customwidget>
  <customwidget>
   <class>IndicatorButton</class>
   <extends>QPushButton</extends>
   <header>indicatorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="uires.qrc"/>
 </resources>
 <connections/>
</ui>
